src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -275,7 +269,7 @@ SscsCaseData convertAppealToSscsCaseData(SyaCaseWrapper appeal) {
     }
 
     private String resolvePostCode(SyaCaseWrapper appeal) {
-        if (appeal.getIsAppointee()) {
+        if (Boolean.TRUE.equals(appeal.getIsAppointee())) {
```

Comment:
Why change this?  It was perfect the way is was before in my opinion.  This simply adds code for the sake of it and adds little value

Reply:
See here - https://riptutorial.com/java/example/2045/using-boolean-in-if-statement

Reply:
Thank you Waqas, I'll take that ;)